cmd/kops/create_secret.go

```diff
@@ -33,6 +33,9 @@ var (
 	# Create an new ssh public key called admin.
 	kops create secret sshpublickey admin -i ~/.ssh/id_rsa.pub \
 		--name k8s-cluster.example.com --state s3://example.com
+
+	kops create secret nodedockercfg -i ~/.docker/config.json \
```

Comment:
So we use `-i` for SSH because that's the arg for ssh public keys.  Except of course it's actually the arg for SSH private keys.  I suspect `-i` was my mistake, and we should probably use `-f`...